policyengine_us/variables/gov/states/id/tax/income/credits/grocery/eligible/id_grocery_credit_qualified_months.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_us.model_api import *
+
+
+class id_grocery_credit_qualified_months(Variable):
+    value_type = float
+    entity = Person
+    label = "Eligible for the Idaho grocery credit"
+    definition_period = YEAR
+    defined_for = StateCode.ID
+    reference = (
+        "https://law.justia.com/codes/idaho/2022/title-63/chapter-30/section-63-3024a/",
+        "https://tax.idaho.gov/wp-content/uploads/forms/EFO00089/EFO00089_12-30-2022.pdf#page=7",
+    )
+
+    def formula(person, period, parameters):
+        eligible_months_sum = 0
+        year = period.start.year
+        for month in range(1, 13):
```

Comment:
```suggestion
        for month in range(1, 1 + MONTHS_IN_YEAR):
```

Reply:
Tried this initially but getting this error:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/66db217d-2f36-4733-ac72-a904c29953f7)